_data/compatibility/trezor.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: Trezor
+internal_url: /en/compatibility/trezor
+logo: /img/compatibility/trezor/trezor.png
+rbf:
+  tested:
+    date: "2018-11-08"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      notification: "na"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "na"
+      shows_original_version: "na"
+  examples:
+    - image: /img/compatibility/trezor/send-screen.png
+      caption: >
+        Sending RBF Transaction - Send transaction screen. Fee options. No RBF options. Transaction not sent RBF.
+    - image: /img/compatibility/trezor/transaction-list-sent.png
+      caption: >
+        Bumping RBF Transaction - Transaction list screen showing sent transaction. No bumping options.
```

Comment:
Remove words 'Bumping RBF Transaction' since the service doesn't allow bumping.

Reply:
Will resolve this as part of the large "prefix" discussion.